Extension/src/Debugger/nativeAttach.ts

```diff
@@ -175,7 +175,12 @@ function execChildProcess(process: string, workingDirectory?: string): Promise<s
             }
 
             if (stderr && stderr.length > 0) {
-                reject(new Error(stderr));
+                if (stderr.indexOf('screen size is bogus') >= 0) {
+                    // ignore this error silently; see https://github.com/microsoft/vscode/issues/75932
+                    // see similar fix for the Node - Debug (Legacy) Extension at https://github.com/microsoft/vscode-node-debug/commit/5298920
+                } else {
+                    reject(new Error(stderr));
+                }
                 return;
```

Comment:
This `return `statement must be inside the `else `statement above it.

Reply:
Okay, thanks. I'll submit a new PR to master for our next release.

Reply:
https://github.com/microsoft/vscode-cpptools/pull/5705